Upstream: yes, bugs.icu-project.org/trac/changeset/35803/icu/trunk/source/test/intltest/dtfmttst.cpp
Reason: test started failing because we are less than 20 years away from 2034/06/04
Source: upstream

Index: icu/trunk/source/test/intltest/dtfmttst.cpp
===================================================================
--- a/test/intltest/dtfmttst.cpp
+++ b/test/intltest/dtfmttst.cpp
@@ -1134,6 +1134,6 @@
         return;
     }
-    parse2DigitYear(fmt, "5/6/17", date(117, UCAL_JUNE, 5));
-    parse2DigitYear(fmt, "4/6/34", date(34, UCAL_JUNE, 4));
+    parse2DigitYear(fmt, "5/6/30", date(130, UCAL_JUNE, 5));
+    parse2DigitYear(fmt, "4/6/50", date(50, UCAL_JUNE, 4));
 }
